tribe-customizations/v2/week/grid-body/events-day/event.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * View: Week View - Event
+ *
+ * This is a template override of the file at:
+ * events-calendar-pro/src/views/v2/week/grid-body/events-day/event.php
+ *
+ * This template override is needed to make the The Events Calendar Pro Extension: Limit Week View Time Range
+ * work with the updated (V2) designs.
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1aiy
+ *
+ * @version 5.0.0
+ *
+ * @var WP_Post $event The event post object with properties added by the `tribe_get_event` function.
+ *
+ * @see tribe_get_event() For the format of the event object.
+ *
+ */
+
+use Tribe__Date_Utils as Dates;
+
+$classes = [ 'tribe-events-pro-week-grid__event' ];
+
+if ( ! empty( $event->is_past ) ) {
+	$classes[] = 'tribe-events-pro-week-grid__event--past';
+}
+
+if ( ! empty( $event->featured ) ) {
+	$classes[] = 'tribe-events-pro-week-grid__event--featured';
+}
+
+/*
+ * Some CSS classes (i.e. vertical position, duration and sequence) have been calculated in the Week View.
+ * Here we add them to the ones that should be applied to the event element.
+ */
+$classes = array_merge( array_values( $classes ), array_values( $event->classes ) );
+$classes = get_post_class( $classes, $event->ID );
+$data_js = [ 'tribe-events-pro-week-grid-event-link', 'tribe-events-tooltip' ];
+
+/**
+ * Get start time in seconds
+ */
+$start_time = Dates::time_between( $event->dates->start->format( 'Y-m-d 0:0:0' ),
```

Comment:
weird formatting - either all one line or put each on its own line and closing `)` on separate line thereafter

Reply:
Not sure what happened here. Original template has it in one line, so that is how I formatted it now.